app/models/BannerTests.scala

```diff
@@ -66,6 +67,6 @@ case class BannerTest(
 
 object BannerTest {
   implicit val customConfig: Configuration = Configuration.default.withDefaults
-  implicit val bannerTestDecoder = Decoder[BannerTest]
-  implicit val bannerTestEncoder = Encoder[BannerTest]
+  implicit val bannerTestDecoder: Decoder[BannerTest] = deriveConfiguredDecoder[BannerTest]
+  implicit val bannerTestEncoder: Encoder[BannerTest] = deriveConfiguredEncoder[BannerTest]
```

Comment:
Previously this used circe's 'automatic' derivation, but I've changed it to 'semi-automatic' derivation.
This is because the compiler gets confused around the new `ChannelTest` encoder/decoder if we use automatic. There's quite a lot of magic happening at compile-time.